Three/include/CGAL/Three/Scene_interface.h

```diff
@@ -168,7 +113,12 @@ class Scene_interface {
   //!
   //!@returns the index of the Item_B
   virtual Item_id selectionBindex() const = 0;
-
+  /*!
+   * \brief isPolyhedronMode determines if the global state is Polyhedron mode or Surface_mesh mode.
+   * @returns true if the global state is Polyhedron mode
+   * @returns false if the global state is Surface_mesh mode
+   */
```

Comment:
That documentation is completely useless. That is similar to:

```
/// Tells is Foo is true or not
bool isFoo() const;
```

The documentation should say what is changed by that global state.